digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala

```diff
@@ -786,22 +801,26 @@ trait LinearAssetOperations {
     }
   }
 
-  def getActiveMaintenanceRoadByPolygon(areaId: Int, typeId: Int): Seq[PersistedLinearAsset] = {
-    val polygon= polygonTools.getPolygonByArea(areaId)
-    val vVHLinkIds = roadLinkService.getLinkIdsFromVVHWithComplementaryByPolygons(polygon)
-    getPersistedAssetsByLinkIds(typeId, vVHLinkIds)
+  def getMunicipalityById (id: Long): Seq[Long]={
```

Comment:
check if this method is used and replace by 'getMunicipalityById'  in municipalityDAO